### PR TITLE
Upmerge 2022.09.27

### PR DESCRIPTION
**Pull request by anangl**
Update the following projects:

- zephyr (see https://github.com/nrfconnect/sdk-zephyr/pull/916)
- mcuboot (see https://github.com/nrfconnect/sdk-mcuboot/pull/215)
- trusted-firmware-m (see https://github.com/nrfconnect/sdk-trusted-firmware-m/pull/68)
- nrfxlib (see https://github.com/nrfconnect/sdk-nrfxlib/pull/838)
- homekit (see https://github.com/nrfconnect/sdk-homekit/pull/445)
- matter (see https://github.com/nrfconnect/sdk-connectedhomeip/pull/125)
- sdk-hostap (see https://github.com/nrfconnect/sdk-hostap/pull/34)
- find-my (see see https://github.com/nrfconnect/sdk-find-my/pull/111)

to align with https://github.com/zephyrproject-rtos/zephyr/commit/b96453f653b674629c617bdbc810c3d9f7916ef4.

test-sdk-nrf: sdk-nrf-pr-8883

### CODE REVIEW

**Review by pdunaj — APPROVED** [review]
Changes look good. @wiminordic , @jumi-nordic was there any our DFU test executed for this PR?

**Review by krish2718 — APPROVED** [review]
LGTM, there is a CI failure in Wi-Fi parts in existing code, I will raise a PR for that separately, but for this PR we should override that check

**Review by Balaklaka — APPROVED** [review]

**Review by MarekPieta — CHANGES_REQUESTED** [review]
nRF Desktop, ML app and Fast Pair sample build fail (dependency loop related to MBEDTLS configuration). It seems like common issue appearing in the NCS. It needs to be fixed to enable further testing.

**Comment by wiminordic** [comment]
@pdunaj no. It can be done manually when we get a green build.

**Comment by simensrostad** [comment]
No idea how I managed to remove the review request for so many people in one setting. The intention was only to remove meg self.

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Review by plskeggs — APPROVED** [review]

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Review by jhn-nordic — APPROVED** [review]
Passes thingy91 nigthly

**Comment by anangl** [comment]
Rebased.

**Comment by anangl** [comment]
Rebased.

**Review by SeppoTakalo — APPROVED** [review]

**Review by rugeGerritsen — APPROVED** [review]

**Comment by anangl** [comment]
Rebased.

**Review by sachinthegreen — APPROVED** [review]

**Review by hermabe — APPROVED** [review]

**Comment by anangl** [comment]
Rebased.

**Review by MarekPieta — APPROVED** [review]
nRF Desktop and nRF Machine learning applications seem to work fine already. Thanks for the fixes

**Comment by anangl** [comment]
Rebased and updated with SHAs for the matter, homekit, and find-my projects.

**Comment by anangl** [comment]
Rebased.

**Review by greg-fer — APPROVED** [review]
Changelog updates look good. Other doc changes mirror code changes.

File: scripts/west_commands/ncs_commands.py [hunk 1]
```diff
@@ -505,11 +505,14 @@ def _name_and_path(project):
      'modules/hal/openisa',
      'modules/hal/qmsi',
      'modules/hal/quicklogic',
+     'modules/hal/renesas',
      'modules/hal/rpi_pico',
      'modules/hal/silabs',
      'modules/hal/stm32',
      'modules/hal/telink',
      'modules/hal/ti',
+     'modules/hal/wurthelektronik',
```
Comment (carlescufi) [inline scripts/west_commands/ncs_commands.py:514]: This must be enabled (it's a sensor HAL), can be done in a follow-up PR. Remember to include it in the `allowlist` of the import.

**Review by carlescufi — APPROVED** [review]

File: scripts/west_commands/ncs_commands.py (hunk 1 above)
Comment (anangl) [reply scripts/west_commands/ncs_commands.py:514]: Addressed in https://github.com/nrfconnect/sdk-nrf/pull/9082.

**Pull request merged**